### PR TITLE
RM-8292: Change WebButton default border color to primary blue

### PR DESCRIPTION
**Pull request by iam-mholle**

### CODE REVIEW

**Comment by iam-mholle** [comment]
@MichaelKetting I used the opportunity to remove the `input.webButton` target - no other rules apply to this selector, am I right to assume that it is obsolete? If not, I'll just change it back.

**Comment by MichaelKetting** [comment]
> @MichaelKetting I used the opportunity to remove the `input.webButton` target - no other rules apply to this selector, am I right to assume that it is obsolete? If not, I'll just change it back.

@iam-mholle Please revert that one. If you set the `WebButton.UseLegacyButton` flag to true, you get the `input` element + the css class.

File: Remotion/Web/Core/Res/Themes/NovaViso/HTML/WebButton.css [hunk 1]
```diff
@@ -6,13 +6,13 @@
...
 
-button.webButton,
-input.webButton
+button.webButton
 {
   min-height: var(--webbutton-min-height);
   padding: 0;
   box-sizing: border-box;
   display: inline-block;
+  border-color: var(--color-primary);
```
Comment (MichaelKetting) [inline Remotion/Web/Core/Res/Themes/NovaViso/HTML/WebButton.css:15]: I think we need to do this in Common.css. The border look&feel is unified for all button elements via Common.css, so the special color applied to buttons needs to be implemented there as well.

**Review by MichaelKetting — CHANGES_REQUESTED** [review]

**Comment by iam-mholle** [comment]
> > @MichaelKetting I used the opportunity to remove the `input.webButton` target - no other rules apply to this selector, am I right to assume that it is obsolete? If not, I'll just change it back.
> 
> @iam-mholle Please revert that one. If you set the `WebButton.UseLegacyButton` flag to true, you get the `input` element + the css class.

That means that `primary`/`supplemental` style is not available on the legacy button?

**Comment by MichaelKetting** [comment]
> That means that primary/supplemental style is not available on the legacy button?

Hmm... yeah, I think we didn't notice that one. Could you please create a ticket for this?

**Review by MichaelKetting — CHANGES_REQUESTED** [review]

**Pull request merged**